Makefile

```diff
@@ -441,6 +441,15 @@ k_quants.o: k_quants.c k_quants.h
 	$(CC) $(CFLAGS) -c $< -o $@
 endif # LLAMA_NO_K_QUANTS
 
+ifndef LLAMA_NO_SKIP_UNUSED_LOGITS
+	CFLAGS   += -DLLAMA_SKIP_UNUSED_LOGITS
```

Comment:
You can't put indented assignments below a build rule like this. The ifdef doesn't make a difference syntactically; this line is treated as part of the k-quants build.

Reply:
Ah thanks, I'd overlooked https://github.com/ggerganov/llama.cpp/commit/4dcd47d71df8ca4edcc31302744bd93f0c31298e (and hadn't merged it well), fixed.